README.md

```diff
@@ -21,3 +40,4 @@ MIT lisence with exceptions:
 - The app icon is designed by [nervouna](https://github.com/nervouna) under MIT license.
 - The menu bar icon is copied from Google's Material Design under Apache 2.0 license.
 - The SMC.swift is copied from [SMCKit project](https://github.com/beltex/SMCKit) under [MIT license](https://github.com/beltex/SMCKit/blob/master/LICENSE).
+- Forked and adjusted by [mrousavy](https://github.com/mrousavy)
```

Comment:
Hi I'd like to add any contributors name in the README file, but this section is for the codes (or assets) which is copied from other project's  LICENSE, so I think this change here is not suitable.

I think we can add another section to show contributor's name.

And I have another project, is using a service to the contributor's name like this: https://github.com/aisk/rust-memcache/blob/master/README.md#contributors , I think you can take a look it's source code and copy it here.